pyhap/accessory_driver.py

```diff
@@ -63,6 +64,13 @@ def __init__(self, accessory, address, port):
              adv_data,
              pubname)
 
+    @property
```

Comment:
I would leave this as a method - my understanding is that `@property` should be for simple computations. Anyways, I don't mind.

Reply:
Done in next commit.